src/modules/Dropdown/Dropdown.js

```diff
@@ -1047,7 +1047,9 @@ export default class Dropdown extends Component {
     } else if (hasValue) {
       _text = _.get(this.getItemByValue(value), 'text')
     }
-
+    if (classes === 'text') {
+      return <div className={classes} role='alert' aria-live='polite'>{_text}</div>
+    }
```

Comment:
I believe `classes === 'text'` is the same as the final return case here.  When none of the above conditions match, the text is rendered as:

```html
<div class="text">File</div>
```

<img src=https://user-images.githubusercontent.com/5067638/27964988-0fcccd8e-62ef-11e7-9275-29cfd157ad50.png width=400 />

Is there a reason not to add `role='alert' aria-live='polite'` to the final return value instead?